ArpSrc/Sequitur/SeqFilterAddOnWindow.cpp

```diff
@@ -318,8 +318,10 @@ SeqFilterAddOnWindow::SeqFilterAddOnWindow(	BRect frame,
 	AddViews();
 	if (!roster) roster = AmFilterRoster::Default();
 	mRoster = roster;
-	AmMultiFilterRoster*	roster = AmMultiFilterRoster::Default();
-	if (roster) roster->AddObserver(this);
+	//HACK to make compile on 64 bit. What the hell s going on here?
```

Comment:
It looks like there were a local variable and a parameter both named roster and one needs to be renamed?
This is (was?) allowed by the standard but the compiler generates a warning now.

Reply:
Removed the comment and commented out line, as it seems to be the right fix for it.